### DISCUSSION TITLE
Post-processing with a single-fold.

### DISCUSSION THREAD

**Discussion by stalhabukhari** [MIC-DKFZ/nnUNet#684] [Q&A] [answered]
Hi @FabianIsensee,

I trained nnUNet on a custom split of the BraTS dataset. I am not interested in an ensemble over 5-folds, and only want *one* trained network. The validation set (online) scores for tumor core and enhancing core are lower than expected, probably because I did not perform post-processing (apart from converting Label 3 to 4). So I am interested in post-processing without training over 5-folds, which is not provided here. Is there a direct way around for this? Can you share the post-processing configuration (`.json` file) you used for the latest BraTS dataset?

P.S.: I did all things necessary before training, i.e., data conversion, planning and pre-processing, etc.

**Comment by FabianIsensee** [comment, marked as answer]
The nnunet postprocessing is not the one I used in Brats, please look at the BraTS2020 conversion script for that. You can just train the 'all' (all here means nnunet trains on all training data) fold, predict the official validation cases with that and apply the same postprocessing to the predictions as I did in the BraTS2020 script - done.
Please double and triple check that the labels are the correct integer values before uploading. Also read up on when the postprocessing must be applied (could be before or after label conversion, I don't remember)
Best,
Fabian

**Reply by stalhabukhari** [reply]
Ah! It worked! Just running `BraTS_data_conversion.py` made a huge difference.

Thank you!
